pkg/controller/authentication/configmap.go

```diff
@@ -45,40 +52,37 @@ func (r *ReconcileAuthentication) handleConfigMap(instance *operatorv1alpha1.Aut
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: consoleConfigMapName, Namespace: instance.Namespace}, consoleConfigMap)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			reqLogger.Error(err, "The configmap ", consoleConfigMapName ," is not created yet")
-			return err
-		} else {
-			reqLogger.Error(err, "Failed to get ConfigMap",  consoleConfigMapName)
+			reqLogger.Error(err, "The configmap ", consoleConfigMapName, " is not created yet")
```

Comment:
@yannizhang2019 why did you remove the else block here

Reply:
that's from go lint